src/mon/Elector.h

```diff
@@ -425,9 +425,6 @@ class Elector {
    * @post  @p participating is true
    */
   void start_participating();
-  /**
```

Comment:
why are you removing the closing tag for this doxygen defgroup? I'm assuming this was a mistake.